### ISSUE TITLE
Centrifuge report not recognized 

### ISSUE THREAD

**Issue by voutcn** [fbreitwieser/pavian#2]
a.t.t.

Below are the first 30 lines of the file `centrifuge_report.tsv` I want to submit to pavian.

```tsv
name	taxID	taxRank	genomeSize	numReads	numUniqueReads	abundance
Buchnera aphidicola	9	species	621119	397	397	0.0
Dictyoglomus	13	genus	1907773	22	22	0.0
Shewanella	22	genus	5140018	5785	5785	0.0
Shewanella putrefaciens	24	species	4749735	64	64	0.0
Myxococcales	29	order	9744470	46	46	0.0
Myxococcaceae	31	family	9924679	2	2	0.0
Myxococcus	32	genus	9885694	29	29	0.0
Cystobacteraceae	39	family	10260756	1	1	0.0
Archangium gephyra	48	species	12489432	74	74	0.0
Polyangiaceae	49	family	13907952	427	427	0.0
Chondromyces crocatus	52	species	11388132	5081	5081	0.0
Sorangium cellulosum	56	species	13907952	111	111	0.0
Lysobacter	68	genus	0	95	95	0.0
Caulobacter	75	genus	4238499	275	275	0.0
Hyphomicrobium	81	genus	3700497	6	6	0.0
Planctomycetales	112	order	6589575	7	7	0.0
Spirochaetaceae	137	family	2844607	8	8	0.0
Borrelia	138	genus	1104217	271	271	0.0
Borreliella burgdorferi	139	species	1329613	96	96	0.0
Spirochaeta	146	genus	3243173	1	1	0.0
Spirochaeta thermophila	154	species	2516433	30	30	0.0
Treponema	157	genus	2625101	148	148	0.0
Leptospira	171	genus	4731147	31	31	0.0
Leptospira biflexa	172	species	3953768	65	65	0.0
Leptospira interrogans	173	species	5395453	154	154	0.0
Leptospira borgpetersenii	174	species	3948532	19	19	0.0
Leptospirillum	179	genus	2432093	11	11	0.0
Azospirillum	191	genus	7223069	14	14	0.0
Azospirillum brasilense	192	species	13978806	152	152	0.0
```

**Comment by fbreitwieser** [comment]
Thanks for trying it out, @voutcn . Pavian does not support the default report format of Centrifuge, which does not include enough taxonomical information. I made changes to make this clear, and also such that Pavian does not fail when an unsupported file is uploaded.

To get the correct (Kraken-style) centrifuge report format, please run:

```
centrifuge-kreport -x IDX CENTRIFUGE_OUT > new_report.csv
```

Note that with `centrifuge-kreport` you can also do some quality filtering, such as `--min-length=N` to require a minimum alignment length.

**Comment by voutcn** [comment]
Thanks @fbreitwieser and it works! Great work!

**Comment by klahmers1** [comment]
I am so glad to find this.  I tried and failed.  

lahmersk-iml12:indices kevinlahmers$ centrifuge-kreport -x p_compressed+h+v prrscentrifuge_report.tsv > newprrs_report.csv
-bash: centrifuge-kreport: command not found

So then I tried:

lahmersk-iml12:indices kevinlahmers$ centrifuge -kreport -x p_compressed+h+v prrscentrifuge_report.tsv > newprrs_report.csv
-k arg must be at least 1

I missed something here.
